### ISSUE TITLE
avoid.overlapping for kpPlotTranscript

### ISSUE THREAD

**Issue by jotsetung** [bernatgel/karyoploteR#11]
Hi Bernat!

I've realized that `kpPlotRegions` has an argument `avoid.overlapping` to stack regions and avoid overlapping them - would be nice if `kpPlotTranscripts` had also the same option.

**Comment by bernatgel** [comment]
Hi Johannes,

Actually, the transcripts do not overlap other transcripts (or at least should not) but the transcript positioning algorithm does not take into account the labels. This is something I've been meaning to add for quite some time, but had no time in the last few months. I'm adding this to the TODO list.

Thanks for the suggestion :)

Bernat

**Comment by dstern** [comment]
It would be great if this issue could be addressed for kpPlotGenes as well. I have two small neighboring genes and the gene labels overlap.

**Comment by bernatgel** [comment]
Yes, thisis one of karyoploteR's most pressing issues. Gene/transcript label overlaps and gene/transcript associated issues. I'll try to work it out in the next few months, although if anyone has any code to contribute, I'll be glad to test and  merge it!
